src/App.scss

```diff
@@ -0,0 +1,25 @@
+%grid {
+  display: grid;
+}
+
+.calculator {
+  @extend %grid;
+
+  grid-template-rows: auto auto;
+  width: max-content;
+  margin: auto;
+
+  .display {
+    justify-self: flex-end;
+  }
+
+  .keyboard {
+    @extend %grid;
+
+    grid-template-columns: repeat(4, 50px);
+
+    .span-two-col {
+      grid-column: span 2;
+    }
+  }
+}
```

Comment:
- [ ] Please make sure to stick with the design, currently, your input field has no border as well. Also, make it a little bigger so that the user can easily view the calculator. :slightly_smiling_face: 

### Your design

![Screenshot from 2021-08-12 17-08-36](https://user-images.githubusercontent.com/22434660/129191133-ae1f351b-a98e-40b8-9a53-12272d9165ba.png)


### Given design

![Screenshot from 2021-08-10 20-48-42](https://user-images.githubusercontent.com/22434660/128893622-b08fc46e-f651-4e05-a86e-7d4bb2c5cefc.png)